src/raygui.h

```diff
@@ -3284,8 +3284,8 @@ Vector2 GuiGrid(Rectangle bounds, const char *text, float spacing, int subdivs)
         if (CheckCollisionPointRec(mousePoint, bounds))
         {
             // NOTE: Cell values must be rounded to int
-            currentCell.x = (int)((mousePoint.x - bounds.x)/spacing);
-            currentCell.y = (int)((mousePoint.y - bounds.y)/spacing);
+            currentCell.x = roundf((mousePoint.x - bounds.x)/spacing);
```

Comment:
Afair, those lines have been already changed in the past because of issues when dealing with floats, the final solution was a cast to `int`. I can't remember which projects were affected by this... in any case I'm merging and I'll see if anything brokes in the future.